leakcanary-deobfuscation-gradle-plugin/src/main/java/com/squareup/leakcanary/deobfuscation/CopyObfuscationMappingFileTask.kt

```diff
@@ -0,0 +1,56 @@
+package com.squareup.leakcanary.deobfuscation
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.GradleException
+import org.gradle.api.tasks.CacheableTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.PathSensitive
+import org.gradle.api.tasks.PathSensitivity
+import org.gradle.api.tasks.SkipWhenEmpty
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+
+@CacheableTask
+open class CopyObfuscationMappingFileTask : DefaultTask() {
+
+  @Input
+  @SkipWhenEmpty
```

Comment:
Sorry, but I see know that we want to crash when the mappingFile or the mergeAssetsDirectory are null or not defined. 
So `SkipWhenEmpty` doesn't make sense.
I understand now that there are 2 ways to do things and we mix them:
* either we always configure a task for all variants and the task can be skipped if it doens't apply. I find this one confusing and making little sense in this case. It also goes against our variant filtering mechanism.
* OR we create tasks only for qualifying variants. We do that via the filtering. And when it is applied, it has to work and actually do copy the file. If anything goes wrong, we crash. In that case, skipping is not an option.
--> We can remove the annotation.